### PR TITLE
Contact list

### PR DESCRIPTION
**Pull request by adrian-ponce-wolox**
## Summary ##
Contact list implementation
* Show a list of contacts with avatar, nick and last message
* Show a separator between contacts

screen shot 2017-10-02 at 3 44 10 pm

### CODE REVIEW

File: wchat/screens/ContactList/components/ContactCell/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            <Text style={styles.message}>Lorem ipsum dolor sit amet...</Text>
+          </View>
+        </View>
+        <View style={styles.separator} />
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```
Comment (IgnacioAbadie) [inline wchat/screens/ContactList/components/ContactCell/index.js]: We extract all styles into a styles.js file at same level that index.js component, then import it to use styles.

Comment (IgnacioAbadie) [reply wchat/screens/ContactList/components/ContactCell/index.js]: Like in react

File: wchat/screens/ContactList/components/ContactCell/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+   flex: 1,
+  },
+  content: {
+   flex: 1,
+   flexDirection: 'row',
+   marginVertical: 12,
+  },
+  avatar: {
+    height: 44,
+    width: 44,
```
Comment (IgnacioAbadie) [inline wchat/screens/ContactList/components/ContactCell/index.js]: We extract into a constant, sizes that are a prop of  a element. So, this image in particular use 44 pixels size height and width, so ill call avatarSize at the top of the styles.js... in this way if you have to set his size property, just change this const

File: wchat/screens/ContactList/components/ContactCell/index.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    marginBottom: 4,
+  },
+  message: {
+    fontSize: 14,
+    textAlign: 'left',
+    color: '#999',
+  },
+  separator: {
+    height: 1,
+    backgroundColor: '#CCC',
```
Comment (IgnacioAbadie) [inline wchat/screens/ContactList/components/ContactCell/index.js]: We are using in line style, so we have the advance that , in the styles.js file, we can import another .js file... Like colors.js  so extrac all color from you app to a colors.js file... And place this colors file into a constants folder :)

**Review by IgnacioAbadie — CHANGES_REQUESTED** [review]
Nice contacts! Jaj 
Just a little changes

File: wchat/screens/ContactList/components/ContactCell/index.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    return (
+      <View style={styles.container}>
+        <View style={styles.content}>
+          <Image style={styles.avatar} source={{uri: this.props.avatar}} />
+          <View style={styles.info}>
+            <Text style={styles.name}>{this.props.username} </Text>
+            <Text style={styles.message}>Lorem ipsum dolor sit amet...</Text>
+          </View>
+        </View>
+        <View style={styles.separator} />
```
Comment (sbalay) [inline wchat/screens/ContactList/components/ContactCell/index.js]: Can't we implement this using border styles?

**Review by sbalay — APPROVED** [review]

**Comment by adrian-ponce-wolox** [comment]
Updated screen shot:
screen shot 2017-10-03 at 11 35 17 am

**Review by IgnacioAbadie — APPROVED** [review]

**Pull request merged**